src/assets/styles/_constants.scss

```diff
@@ -0,0 +1,24 @@
+/*Fonts*/
+$font: 'Inter';
+$font-size: 16px;
+$line-height: 24px;
+
+/* Colors */
+:root{
+  --text-primary-light: #0b0b0f;
+  --text-primary-dark: #ffffff;
+  --text-secondary: #898ca9;
+  --background-primary-dark: #0b0b0f;
+  --background-primary-light: #f1eeee;
+  --base-dark: #1a1b23;
+  --base-light: #ffffff;
+}
+
+$accent: #b982ff;
+$highlight: #FF9800;
+$gradient-primary: linear-gradient(90deg, #18c8ff 0%, #933ffe 100%);
+$gradient-secondary: linear-gradient(90deg, #933ffe 0%, #18c8ff 100%);
+
+/* Break points */
+$small-mobile: 321px;
+$mobile: 660px;
```

Comment:
How did you decide which variables should be native css and which should be scss variables?